frontend/src/containers/DocList.js

```diff
@@ -23,13 +23,6 @@ const EMPTY_OBJECT = {};
  * @extends Component
  */
 class DocList extends Component {
-  shouldComponentUpdate(nextProps) {
```

Comment:
If we remove the checks from the shouldComponentUpdate wouldn't this impact the re-renders ? Could be this tuned up to allow the re-renderings in sync with the quickActions store updates ?